README.md

```diff
@@ -1,2 +1,51 @@
-# ovirt-ansible-disaster-recovery
-part of the ovirt-ansible project 
+oVirt Disaster Recovery
+=========
+
+The `oVirt.disaster-recovery` role responsible to manage the disaster recovery scenarios in oVirt.
+
+Requirements
+------------
+
+ * Ansible version 2.4 or higher
+ * Python SDK version 4.2.1-1 or higher
```

Comment:
Use here just 4.2.1, it's version which is possible to install from pip.

Reply:
done